lib/src/base-styles/_nx-card.scss

```diff
@@ -6,176 +6,74 @@
  */
 
 .nx-card-container {
+  column-gap: $nx-spacing-md;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
   margin: $nx-spacing-md 0;
+  row-gap: $nx-spacing-md;
+}
+
+.nx-card-container--no-wrap {
+  flex-wrap: nowrap;
 }
 
 .nx-card {
+  @include container-vertical;
   background-color: var(--nx-component-background-color);
   border-radius: $nx-border-radius;
   box-sizing: border-box;
-  margin-bottom: $nx-spacing-md;
-  padding: $nx-spacing-l $nx-spacing-xxl;
+  display: flex;
+  flex-direction: column;
+  padding: $nx-spacing-l;
+  text-align: center;
+  min-width: 230px;
+  max-width: 330px;
 }
 
-.nx-card__call-out {
-  @include bold();
-
-  font-size: 32px;
-
-  .nx-icon--card-call-out {
-    font-size: 48px;
-  }
-
-  .nx-icon--card-call-out-xl {
-    font-size: 64px;
-  }
+.nx-card__header {
+  @include container-vertical;
+  margin-bottom: $nx-spacing-md;
 }
 
-.nx-card-container--row {
+.nx-card__content {
```

Comment:
I think this should have `container-vertical`.  Look at the margins in the nx-read-only example to see why

Reply:
> I think this should have `container-vertical`. Look at the margins in the nx-read-only example to see why

Yes: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/14b0b0f44eacfc4d9dfa3f9b703f2a046ee0ef63